src/routes/+page.svelte

```diff
@@ -25,34 +25,92 @@
     </div>
 </section>
 
-<section class="px-6 lg:px-32 pt-24">
+<section class="px-6 lg:px-32 pt-12 pb-8">
     <div class="border-b border-gray-500 py-12 mb-8 lg:flex lg:justify-between">
-        <h2 class="flex items-center text-6xl font-interSemi tracking-tight pb-4">
+        <h2
+            class="flex items-center text-6xl font-interSemi tracking-tight pb-4"
+        >
             <span class="pr-4"><Values /></span> Beliefs
         </h2>
-        <p class="text-xl font-interMedium max-w-[580px]">
+        <p class="text-2xl font-interMedium max-w-[580px]">
             Crypto has the potential to give people across the world almost
             unimaginable levels of freedom and autonomy.
         </p>
     </div>
     <div class="border-b border-gray-500 py-12 mb-8 lg:flex lg:justify-between">
-        <h2 class="flex items-center text-6xl font-interSemi tracking-tight pb-4">
+        <h2
+            class="flex items-center text-6xl font-interSemi tracking-tight pb-4"
+        >
             <span class="pr-4"><Values /></span> Vision
         </h2>
-        <p class="text-xl font-interMedium max-w-[580px]">
-            We envision a world where everyone has individual sovereignty, economic freedom and ownership of the infrastructure that supports them.
+        <p class="text-2xl font-interMedium max-w-[580px]">
+            We envision a world where everyone has individual sovereignty,
+            economic freedom and ownership of the infrastructure that supports
+            them.
         </p>
     </div>
     <div class="border-b border-gray-500 py-12 mb-8 lg:flex lg:justify-between">
-        <h2 class="flex items-center text-6xl font-interSemi tracking-tight pb-4">
+        <h2
+            class="flex items-center text-6xl font-interSemi tracking-tight pb-4"
+        >
             <span class="pr-4"><Values /></span> Mission
         </h2>
-        <p class="text-xl font-interMedium max-w-[580px]">
-            Accelerate Ethereum adoption, because for this vision to materialize, crypto must be built on a truly decentralized and permissionless base layer.
+        <p class="text-2xl font-interMedium max-w-[580px]">
+            Accelerate Ethereum adoption, because for this vision to
+            materialize, crypto must be built on a truly decentralized and
+            permissionless base layer.
         </p>
     </div>
 </section>
 
+<section class="px-6 lg:px-32 pt-12">
+    <div class="pb-16">
```

Comment:
Prefer gap over padding for creating space between elements. Despite creating the same effect, what you want is a gap between the elements,  not a space around the element